src/Sylius/Behat/Context/Setup/OrderContext.php

```diff
@@ -160,6 +160,19 @@ public function thereIsCustomerThatPlacedOrder(CustomerInterface $customer, $ord
     }
 
     /**
+     * @Given the customer :customer have already placed an order :orderNumber
```

Comment:
After the change (look for all orders, not only paid ones) from our morning discussion it will be possible :)